core/templates/dev/head/components/fallback_editor_directive.html

```diff
@@ -14,10 +14,10 @@
           Once the learner has made <[trigger.customization_args.num_submits.value]>
           unsuccessful attempt<span ng-if="trigger.customization_args.num_submits.value !== 1">s</span>:
 
-          <span ng-if="isEditable"
-            class="glyphicon glyphicon-pencil oppia-editor-edit-icon pull-right"
-            title="Edit Fallback Condition">
-          </span>
+          <i ng-if"isEditable"
+            class="material-icons oppia-editor-edit-icon pull-right" 
```

Comment:
Nit: the current HTML "style", insofar as we have one, is to align the attribute names if a tag spans more than one line. (I.e., in this case, indent this line by one more space.) Could you please do that for this tag, and for other tags in the PR?